neuraxle/steps/loop.py

```diff
@@ -25,12 +25,12 @@
 import copy
 from typing import List, Any
 
-from neuraxle.base import MetaStepMixin, BaseStep, DataContainer, ExecutionContext, ResumableStepMixin
+from neuraxle.base import MetaStepMixin, BaseStep, DataContainer, ExecutionContext, ResumableStepMixin, HandlerMixin
 from neuraxle.data_container import ListDataContainer
 from neuraxle.hyperparams.space import HyperparameterSamples, HyperparameterSpace
 
 
-class ForEachDataInput(ResumableStepMixin, MetaStepMixin, BaseStep):
+class ForEachDataInput(HandlerMixin, ResumableStepMixin, MetaStepMixin, BaseStep):
```

Comment:
should we add here also something that if it's `transform` that it uses `_handle_transform` automatically?